extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -1048,20 +1061,16 @@ static public function doAfterPost( Status $status, $article, $parentId = 0 ) {
 				wfGetDB( DB_MASTER )->commit();
 				break;
 			default:
-				$userId = $wgUser->getId();
 				$text  = false;
 				$error = true;
 
 				$message = wfMessage( 'article-comments-error' )->escaped();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "article-comments-error" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)